sql/moz-fx-data-experiments/monitoring/experimenter_experiments_v1/query.py

```diff
@@ -145,7 +145,9 @@ def from_dict(cls, d) -> "ExperimentV6":
         converter = cattr.Converter()
         converter.register_structure_hook(
             datetime.datetime,
-            lambda num, _: datetime.datetime.fromisoformat(num.replace("Z", "+00:00")),
+            lambda num, _: pytz.utc.localize(
+                datetime.datetime.fromisoformat(num.replace("Z", "+00:00"))
+            ),
```

Comment:
when i test this locally I get `ValueError: Not naive datetime (tzinfo is already set)`

Reply:
to reproduce:
```python
num = datetime.datetime.utcnow().isoformat() + "Z"
pytz.utc.localize(datetime.datetime.fromisoformat(num.replace("Z", "+00:00")))
```